src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -146,12 +141,7 @@ public function apply(QueryBuilder $queryBuilder, string $resourceClass, string
                 }
```

Comment:
maybe add a `return` here to just do a `if` after and avoid an remove an indent level